### ISSUE TITLE
Postman latency does not match server latency

### ISSUE THREAD

**Issue by terencechow** [a85/POSTMan-Chrome-Extension#3882]
## App Details:
```
Postman -> About postman <--- is greyed out so I can't provide postman details
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: not sure
 2. Expected behaviour:
 3. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app):
 4. Screenshots (if applicable)

1. I am running a kubernetes cluster with 2 restify servers and 1 mysql service. These are run locally with minikube using virtualbox. 1 server is my API and another is my backend which speaks to the local mysql database.

In my restify server I use the audit logger which tells me latency times of a request. Looking at the code of restify, the audit logger tracks the latency of a request from when it is received to the moment it is sent back. Therefore it isn't the moment the Client has received the full response, but the time from when the API has received a request, to when it has started to send a response back.

I'm using Postman as my client. When I use Postman to make a request to my API it says the latency is around 300 ms. However my audit logger from the API says it is only 200ms. 100ms seems like a big difference between my API and Postman, considering this is all within the same computer.

In short, I'm seeing an addition 100ms - 200ms for each request on the same computer which seems extremely odd. What could be the cause of this?

2. Expect the latency of Postman to match the latency of my API audit logger

**Comment by kamalaknn** [comment]
Hi @terencechow 

I tried to recreate this scenario by setting up a small node server locally and measuring the response time on Postman. 

I saw an initial 35ms - 50ms extra time in Postman that includes the socket creation, DNS lookup. It then reduced to 5ms - 7ms once the TCP level socket caching and DNS lookup caching(OS level) kicked in.

Could you verify that your measurement is accurate and if there are any other subsystems involved in your request flow?

**Comment by terencechow** [comment]
I decided to remove all kubernetes related things and just have an express server.

index.js
```
let data = []
for (var i =0; i < 1000; i++){
  data.push({
    'someobject': 'something'
  })
}

var express = require('express')
var server = express()

server.get('/test', function(req,res,next){
  res.send(data)
})

server.listen(8080, function () {
  console.log('ready on %s', server.url);
});

```

1. `npm install express`
2. `node index.js`
3. with postman make a get request to `localhost:8080/test`
4. postman reports 250-400ms
5. Repeat step 3. Postman always reports 250-400ms.
6. run `ab -c 20 -n 1000 localhost:8080/test`
7. AB reports 99% takes less than 20 ms. Longest one took 17 ms.

**Comment by kamalaknn** [comment]
@terencechow 

Are you using the Chrome App or the Postman Native app? You can find it from the "How can I tell if I’m on the Postman Chrome app or Postman native app?" section in this [blog](http://blog.getpostman.com/2017/03/14/going-native/).

**Comment by terencechow** [comment]
I am on the Postman Chrome app

**Comment by kamalaknn** [comment]
Ah, that explains it. 

Could you try downloading the Postman Native Apps and check out the response time. The Native apps allow us more control over the request sending flows so we can get better results for response times.
